### ISSUE TITLE
Bangbros metadata broken

### ISSUE THREAD

**Issue by chempke** [PAhelper/PhoenixAdult.bundle#182]
find scene but don't any meta except summary. 

log: 
```
2019-05-07 22:07:19,259 (4310) :  DEBUG (networking:166) - Requesting 'https://bangbros.com/video118211/big-booty-white-girl-fucked-outdoors'
2019-05-07 22:07:20,036 (4310) :  DEBUG (networking:166) - Requesting 'https://bangbros.com/search/pwg13145'
2019-05-07 22:07:20,869 (4310) :  CRITICAL (core:574) - Exception in the update function of agent named 'PhoenixAdult', called with guid 'com.plexapp.agents.phoenixadult://_video118211_big-booty-white-girl-fucked-outdoors|126?lang=en' (most recent call last):
  File "C:\Program Files (x86)\Plex\Plex Media Server\Resources\Plug-ins-4610c6e8d\Framework.bundle\Contents\Resources\Versions\2\Python\Framework\api\agentkit.py", line 1076, in _update
    agent.update(obj, media, lang, **kwargs)
  File "C:\Users\dzemk\AppData\Local\Plex Media Server\Plug-ins\PhoenixAdult.bundle\Contents\Code\__init__.py", line 877, in update
    metadata = PAsearchSites.siteBangBros.update(metadata,siteID,movieGenres,movieActors)
  File "C:\Users\dzemk\AppData\Local\Plex Media Server\Plug-ins\PhoenixAdult.bundle\Contents\Code\siteBangBros.py", line 43, in update
    date_object = datetime.strptime(releaseDate, '%b %d, %Y')
  File "C:\Program Files (x86)\Plex\Plex Media Server\python27.zip\_strptime.py", line 332, in _strptime
    (data_string, format))
ValueError: time data '                                            Jul 25, 2014                                                        ' does not match format '%b %d, %Y'

2019-05-07 22:07:20,871 (4310) :  DEBUG (model:229) - Serializing to C:\Users\dzemk\AppData\Local\Plex Media Server\Metadata\Movies\d\f9c68de2d18368515816d97170b79f83a94aaf7.bundle\Contents\com.plexapp.agents.phoenixadult\Info.xml
2019-05-07 22:07:20,875 (4310) :  DEBUG (runtime:88) - Sending packed state data (596 bytes)
2019-05-07 22:07:20,875 (4310) :  DEBUG (runtime:924) - Response: [200] str, 16 bytes

```

**Comment by claygoldfinch** [comment]
Fixed in PR #181

**Comment by d2dyno** [comment]
Confirmed, @claygoldfinch update fixes the issue. Thanks clay!
